phageAPI/addspacerandrepeat.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+'''
+Issue #120 - testing
+Author: Ahmed
+
+Usage:
+python addspacerandrepeat.py
+
+'''
+
+
+import os
+import sys
+from Bio import SeqIO
+
+spacerinput = os.listdir('../data/spacers') # these are multiple files
```

Comment:
We don't need to use the `/data/spacers` files - everything is in the `spacerdatabase.txt` file.